app/assets/stylesheets/primary_source_sets.css

```diff
@@ -18,6 +18,12 @@
   margin-top: 2em;
 }
 
+#osd-viewer {
+  width: 700px;
+  height: 485px;
+  background-color: black;
+}
```

Comment:
I noticed that on the `source#show` page, the viewer isn't taking up the whole space to the left of the sidebar.  I tried changing the style (above) to `width: 100%; height: 100%` and that worked for me.